src/libPMacc/test/memory/HostBufferIntern/setValue.hpp

```diff
@@ -47,7 +47,7 @@ struct setValueTest
             const Data value = 255;
             hostBufferIntern.setValue(value);
 
-            PMACC_AUTO( ptr, hostBufferIntern.getPointer( ) );
```

Comment:
`PMACC_AUTO`: this part of the code seems not to be covered by the compile suite